config/samples/resources/cloudbuildworkerpool/workerpool-with-default-network/cloudbuild_v1beta1_cloudbuildworkerpool.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: cloudbuild.cnrm.cloud.google.com/v1beta1
+kind: CloudBuildWorkerPool
+metadata:
+  name: cloudbuildworkerpool-sample
```

Comment:
Sample name should be something like `cloudbuilderworkerpool-sample-default-network`.
Naming convention here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#sample-naming-convention

Reply:
I like the idea of uniforming the naming to give cloud users more coherent samples. But it would be too much work to push this trend via PR reviews and not intuitive enough for contributors. We shall do something like ./tests/apichecks to make this more automatic.